pegasus/sites.v3/code.org/public/css/hoc-banner.scss

```diff
@@ -86,8 +91,9 @@
   }
 
   h2 {
-    margin-top: 0;
+    margin: 0 auto 1rem;
     font-family: var(--main-font);
+    font-size: 1.25rem;
 
     @media screen and (max-width: $width-md) {
       font-size: 1.25em;
```

Comment:
If `font-size: 1.25em;` is being placed outside this @media tag, could it be removed here?

Reply:
Great catch! Updated 🙌